### PR TITLE
added cartmanagement component

### PR DESCRIPTION
**Pull request by MonikaJhingan**
review needed!!!

### CODE REVIEW

File: CartMgmt/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --white-color: #fff;
+  --grey-color: #808080;
+  --peach-color:rgb(241, 203, 219);
```
Comment (bajajcodes) [inline CartMgmt/Cart/cart.css:11]: Use consistent value type for colors either of the one.

File: CartMgmt/Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+}
+.navigation {
+  margin: 0rem;
```
Comment (bajajcodes) [inline CartMgmt/Cart/cart.css]: Can use `0` instead of `0 rem`

File: CartMgmt/Cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  border-top: 1px solid black;
+}
+.order-btn {
+  width: 50%;
+  border-radius: 3rem;;
+}
+.btn-link {
+  width: 20rem;
+  height: 1.9rem;
+  font-size:2.rem;
```
Comment (bajajcodes) [inline CartMgmt/Cart/cart.css]: Here's syntax error.

File: CartMgmt/Cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
```
Comment (bajajcodes) [inline CartMgmt/Cart/cart.css:2]: These lines of code can be part of reset or base css.

File: CartMgmt/Cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  margin-right: 30rem;
+  margin-top: 20rem;
+  padding: 3rem;
+}
+
+.btn-shopping-cart:hover {
+  color: black;
+}
+
+table {
```
Comment (bajajcodes) [inline CartMgmt/Cart/cart.css:48]: It's good to use wrapper instead of applying styling on element directly.

**Review by bajajcodes — COMMENTED** [review]
- Define commonly used values in root of css and use as variables.
- Format code to make it look clean and consistent.
- Classes names are meaningful.

File: CartMgmt/Cart/cart.css (hunk 1 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css:11]: ok will keep in mind.

File: CartMgmt/Cart/cart.css (hunk 2 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css]: done

File: CartMgmt/Cart/cart.css (hunk 4 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css:2]: Will look into it.

File: CartMgmt/Cart/cart.css (hunk 5 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css:48]: _ok _

File: CartMgmt/Cart/cart.css (hunk 3 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css]: done

**Review by sharmivijay — APPROVED** [review]
Try to add the colors in the variables file
use a class name that would define the use case.
code looks neat

File: CartMgmt/Cart/cart.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+          </table>
+          <hr class="someClass" />
+
+          <table>
+            <tr>
+              <td class="font-medium">Total Price</td>
+              <td class="font-medium">Rs 2026</td>
+            </tr>
+          </table>
+
```
Comment (aamrapaliwandhre) [inline CartMgmt/Cart/cart.html:121]: remove this extra space

**Review by aamrapaliwandhre — APPROVED** [review]
everything looks fine...good to merge

File: CartMgmt/Cart/cart.html (hunk 6 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.html:121]: ok

**Pull request merged**